doc/classes/OS.xml

```diff
@@ -773,6 +773,65 @@
 				On macOS (sandboxed applications only), this function clears list of user selected folders accessible to the application.
 			</description>
 		</method>
+		<method name="send_notification">
+			<return type="int" enum="Error" />
+			<param index="0" name="title" type="String" />
+			<param index="1" name="message" type="String" />
+			<param index="2" name="callback" type="Callable" default="Callable()" />
+			<param index="3" name="icon" type="Variant" default="null" />
+			<param index="4" name="duration" type="int" default="-1" />
+			<param index="5" name="datetime_to_send" type="Dictionary" default="{}" />
+			<description>
+				Sends a desktop notification to the user with the given title and message.
+
```

Comment:
Every new line creates a "paragraph" as far as the class reference is concerned. Empty new lines are not necessary and will likely cause the static checks to fail.